### ISSUE TITLE
Bad words filter

### ISSUE THREAD

**Issue by Str00pwafel** [Gravebot/Gravebot#175]
I would LOVE to have a bad word filter on this bot. I've tried many bots and this is the most helpfull for our server, since we play league, and it's so easy to use. Only thing I miss is a bad words filter!

**Comment by dustinblackman** [comment]
Do you mean to filter out other people swearing and deleting messages?

**Comment by Str00pwafel** [comment]
Sure do! As far as I'm aware this isn't already in right?

**Comment by dustinblackman** [comment]
Nope! We won't be able to implement it with the current design of Gravebot as we'd have to analyze the entire message of every message that Gravebot receives, and with over 130k users that's a lot. We have plans to setup clustering in the future to make this more viable, then we can look in to implementing this. :)

**Comment by Str00pwafel** [comment]
Got it! Too bad for now then. Will have to wait for that or find an alternative for that only. Thanks for replying, and best of luck towards future development!

**Comment by Ferg00** [comment]
Str00pwafel - if you're still looking for that, [NadekoBot ](https://github.com/Kwoth/NadekoBot) has that feature
